### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by VledicFranco**

### CODE REVIEW

File: src/dev.html [hunk 1]
```diff
@@ -12,7 +12,14 @@
 <div id="app"></div>
 <script src="/app.js"></script>
 <script>
-      Elm.Main.init({node: document.getElementById("app"), flags: {services: {"kratia": ""}}});
+    Elm.Main.init({
+        node: document.getElementById("app"), 
+        flags: {
+            services: {
+                kratia: "http://localhost:8080"
```
Comment (kolov) [inline src/dev.html]: This is good for loal development but will not work if you deploy it. Besides, it needs you to fix CORS headers unlike the proxy setup.

Comment (kolov) [reply src/dev.html]: Here it should be "", maybe with a comment to change it for local development

File: src/Main.elm [hunk 2]
```diff
@@ -77,20 +71,35 @@ subscriptions model =
...
+            ( { model | modalVisibility = Modal.hidden } , Cmd.none )
 
+        ShowModal -> 
+            ( { model | modalVisibility = Modal.shown } , Cmd.none )
+
+        Registered (Ok cred) -> 
+            ( { model | maybeCred = Just cred } , Cmd.none )
+
+        Registered (Err err) -> 
+            ( model, Cmd.none )
```
Comment (kolov) [inline src/Main.elm:96]: loading = false

File: src/Main.elm [hunk 3]
```diff
@@ -77,20 +71,35 @@ subscriptions model =
...
+        NavMsg state -> 
+            ( { model | navState = state } , Cmd.none )
+
+        CloseModal -> 
+            ( { model | modalVisibility = Modal.hidden } , Cmd.none )
 
+        ShowModal -> 
+            ( { model | modalVisibility = Modal.shown } , Cmd.none )
+
+        Registered (Ok cred) ->
```
Comment (kolov) [inline src/Main.elm:92]: loading = false

**Review by kolov — APPROVED** [review]

File: src/Main.elm (hunk 3 above)
Comment (VledicFranco) [reply src/Main.elm:92]: will fix on next one :) thanks

File: src/Main.elm (hunk 2 above)
Comment (VledicFranco) [reply src/Main.elm:96]: will fix on next one :) thanks

**Pull request merged**